tools/devops/automation/templates/governance/stage.yml

```diff
@@ -0,0 +1,95 @@
+# contains the stage used to run all governance related jobs
+
+parameters:
+- name: isPR
+  type: boolean
+
+- name: repositoryAlias
+  type: string
+  default: self
+
+- name: commit
+  type: string
+  default: HEAD
+
+- name: stageDisplayNamePrefix
+  type: string
+  default: ''
+
+stages:
+- stage: governance_checks
+  displayName: '${{ parameters.stageDisplayNamePrefix }}Governance Checks'
+  dependsOn: [ configure_build ]
+
+  jobs:
+  - job: apiscan
+    displayName: 'APIScan:' 
+    pool:
+      vmImage: windows-latest
+
+    variables:
+      INCLUDE_DOTNET_IOS: $[ stageDependencies.configure_build.configure.outputs['configure_platforms.INCLUDE_DOTNET_IOS'] ]
+      INCLUDE_DOTNET_TVOS: $[ stageDependencies.configure_build.configure.outputs['configure_platforms.INCLUDE_DOTNET_TVOS'] ]
+      INCLUDE_DOTNET_MACCATALYST: $[ stageDependencies.configure_build.configure.outputs['configure_platforms.INCLUDE_DOTNET_MACCATALYST'] ]
+      INCLUDE_DOTNET_MACOS: $[ stageDependencies.configure_build.configure.outputs['configure_platforms.INCLUDE_DOTNET_MACOS'] ]
+
+    strategy:
+      matrix:
+        iOS:
+          INCLUDE: $(INCLUDE_DOTNET_IOS)
+          PLATFORM: iOS
+          SECRET: $(ApiScanSecretiOS)
+          VERSION: 17.0
```

Comment:
The VERSION field doesn't seem to be used (which is probably good, because it's outdated already... should be 17.2).

Reply:
@rolfbjarne it will be used, but we will get it from the config step. It is a place holder atm. APIScan takes the version of the product as a parameter so that we can identify the actual problem.